### ISSUE TITLE
Description of images and thumbnails in alt text

### ISSUE THREAD

**Issue by caseyedavis12** [WGBH/openvault3#645]
A short description of all images and thumbnails should be placed in alt text.

**Comment by jasoncorum** [comment]
@caseyedavis12, I'm thinking we do the same thing for search results as specified for AAPB here: https://github.com/WGBH/AAPB2/issues/1083.  Let me know if you think we should handle search results differently on OV than AAPB.

**Comment by afred** [comment]
👍 for consistency.

**Comment by caseyedavis12** [comment]
@jasoncorum, sounds good to me!
